recipe/meta.yaml

```diff
@@ -26,12 +26,15 @@ source:
     folder: pyqtwebengine
 
 build:
-  number: 0
+  number: 1
+  skip: true  # [win and py27]
   skip: true  # [ppc64le]
   run_exports:
     {{ pin_subpackage('pyqt', max_pin='x.x') }}
 
 requirements:
+  run_constrained:                                            # [osx]
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
```

Comment:
I wonder if it is this straightforward to standardize, is there a way to move this into something like `conda-forge-pinning`?

Reply:
Not that I could think of unless we change the condition to be `run` instead of `run_constrained` which forces the user to use `conda>=4.8`